xbmc/windowing/windows/WinEventsWin32.cpp

```diff
@@ -667,6 +667,25 @@ LRESULT CALLBACK CWinEventsWin32::WndProc(HWND hWnd, UINT uMsg, WPARAM wParam, L
         m_pEventFunc(newEvent);
       }
       return(0);
+    case WM_DISPLAYCHANGE:
```

Comment:
you should also update the list of supported resolutions. because it may be a new display which may not support some or all resolutions from old display.